src/http/sr-modules-operators/sr-modules-operators.e2e-spec.ts

```diff
@@ -0,0 +1,374 @@
+        await moduleStorageService.upsert(curatedModule, 1);
+      });
+
+      afterAll(async () => {
+        await cleanDB();
+      });
+
+      it('should return all operators that satisfy the request', async () => {
+        // Get all operators without filters
+        const resp = await request(app.getHttpServer()).get(`/v1/modules/${dvtModule.id}/operators`);
+
+        expect(resp.status).toEqual(200);
+        expect(resp.body.data.operators).toBeDefined();
+        expect(resp.body.data.operators).toEqual(expect.arrayContaining([operatorOneDvt, operatorTwoDvt]));
+
+        expect(resp.body.meta).toEqual({
+          elBlockSnapshot: {
+            blockNumber: elMeta.number,
+            blockHash: elMeta.hash,
+            timestamp: elMeta.timestamp,
+          },
+        });
+
+        const resp2 = await request(app.getHttpServer()).get(`/v1/modules/${curatedModule.id}/operators`);
+
+        expect(resp2.status).toEqual(200);
+        expect(resp2.body.data.operators).toEqual(expect.arrayContaining([operatorOneCurated, operatorTwoCurated]));
+
+        expect(resp2.body.meta).toEqual({
+          elBlockSnapshot: {
+            blockNumber: elMeta.number,
+            blockHash: elMeta.hash,
+            timestamp: elMeta.timestamp,
+          },
+        });
+      });
+
+      it('should return 404 if module was not found', async () => {
+        // Get all operators without filters
+        const resp = await request(app.getHttpServer()).get('/v1/modules/777/operators');
+
+        expect(resp.status).toEqual(404);
+        expect(resp.body).toEqual({
+          error: 'Not Found',
+          message: 'Module with moduleId 777 is not supported',
+          statusCode: 404,
+        });
+      });
+
+      it('should return 400 error if module_id is not a contract address or number', async () => {
+        const resp = await request(app.getHttpServer()).get(`/v1/modules/sjdnsjkfsjkbfsjdfbdjfb/operators`);
+        expect(resp.status).toEqual(400);
+        expect(resp.body).toEqual({
+          error: 'Bad Request',
+          message: ['module_id must be a contract address or numeric value'],
+          statusCode: 400,
+        });
+      });
+    });
+
+    describe('too early response case', () => {
+      beforeEach(async () => {
+        await cleanDB();
+      });
+
+      afterEach(async () => {
+        await cleanDB();
+      });
+
+      it('should return too early response if there are no meta', async () => {
+        // lets save operators
+        await operatorsStorageService.save(operators);
+        await moduleStorageService.upsert(curatedModule, 1);
+
+        const resp = await request(app.getHttpServer()).get(`/v1/modules/${curatedModule.id}/operators`);
+        expect(resp.status).toEqual(425);
+        expect(resp.body).toEqual({ message: 'Too early response', statusCode: 425 });
+      });
+    });
+  });
+
+  describe('The /modules/:module_id/operators/:operator_id', () => {
```

Comment:
do we have a test to get data by module address?

Reply:
added /* eslint-disable @typescript-eslint/no-unused-vars */ 

because of 
```
lass KeysRegistryServiceMock {
    async update(moduleAddress, blockHash) {
      return;
    }
  }
  ```

Reply:
I meant that we have tests for `module.id` but not for `module.address`, although both scenarios are supported in the code

Reply:
confused conversation , sorry 
added test yesterday for moduleAddress